tools/mo/openvino/tools/mo/front/sub.py

```diff
@@ -42,4 +42,6 @@ def sub_to_add_replacement(sub: Node):
 
     def find_and_replace_pattern(self, graph: Graph):
         for sub in graph.get_op_nodes(op='Sub'):
+            if sub.has('dont_replace') and sub['dont_replace']:
```

Comment:
please leave a comment here about quantization and rename attribute name to smth like `zero_point_sub`

Reply:
done